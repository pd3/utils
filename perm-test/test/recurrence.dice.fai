1	6	52	60	61

