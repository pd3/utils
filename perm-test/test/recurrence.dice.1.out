SEED	1587538882
NITER	1.000000e+05
# TEST:
#     - gene .. the region label
#     - nObserved .. number of hits observed in input data in this gene
#     - nSimFewer .. number of simulations with fewer hits in the gene than observed in the data
#     - nSimEqual .. as above, but with the same number of hits than observed
#     - nSimExceeded .. as above, but with the more hits than observed
# DIST:
#     - gene
#     - nSimHits .. the number of simulations with 0,1,2,etc. hits in the gene
# [1]TEST	[2]gene	[3]nObserved	[4]nSimFewer	[5]nSimEqual	[6]nSimExceeded
TEST	side1	2	97227	2773	0
TEST	side2	0	0	69359	30641
TEST	side3	0	0	69367	30633
TEST	side4	0	0	69277	30723
TEST	side5	0	0	69670	30330
TEST	side6	0	0	69428	30572
# [1]DIST	[2]gene	[3-]nSimHits (0,1,etc. hits)
DIST	side1	69470	27757	2773
DIST	side2	69359	27958	2683
DIST	side3	69367	27848	2785
DIST	side4	69277	27933	2790
DIST	side5	69670	27563	2767
DIST	side6	69428	27799	2773
